src/contact.ts

```diff
@@ -718,20 +661,20 @@ export class Contact implements Sayable {
     return Contact.pool[id]
   }
 
+  public async say(text: string)
+
+  public async say(mediaMessage: MediaMessage)
+
   /**
-   * Say `content` to Contact
-   *
-   * @param {string} content
-   * @returns {Promise<void>}
+   * Send Text or Media File to Contact.
    *
+   * @param {(string | MediaMessage)} textOrMedia
+   * @returns {Promise<boolean>}
    * @example
-   * ```ts
+   * const contact = await Contact.find({name: 'lijiarui'})         // change 'lijiarui' to any of your contact name in wechat
    * await contact.say('welcome to wechaty!')
-   * ```
+   * await contact.say(new MediaMessage(__dirname + '/wechaty.png') // put the filePath you want to send here
    */
-  public async say(text: string)
```

Comment:
Why you deleted the overloading function declaration?

They should be kept because those are for TypeScript IDE Intelligence.